schedule.js

```diff
@@ -11,32 +11,26 @@ const colorOfTeam = {
   'World Ventures': 'world'
 }
 
-var games = [
-    {date: "Sept 1st", time: "12:45", away: "Pina Chama", home: "Scorpions"},
-    {date: "Sept 1st", time: "14:45", away: "Papa Spice", home: "World Ventures"},
-    {date: "Sept 8th", time: "12:45", away: "Scorpions", home: "World Ventures"},
-    {date: "Sept 8th", time: "14:45", away: "Pina Chama", home: "Papa Spice"},
-    {date: "Sept 15th", time:  "12:45", away: "World Ventures", home: "Pina Chama"},
-    {date: "Sept 15th", time:  "14:45", away: "Papa Spice", home: "Scorpions"},
-    {date: "Sept 20th", time:  "12:45", away: "Pina Chama", home: "Papa Spice"},
-    {date: "Sept 20th", time:  "14:45", away: "World Ventures", home: "Scorpions"},
-    {date: "Oct 13th", time: "12:45", away: "Scorpions", home: "Papa Spice"},
-    {date: "Oct 13th", time: "14:45", away: "Pina Chama", home: "World Ventures"},
-    {date: "Oct 20th", time: "12:45", away: "World Ventures", home: "Papa Spice"},
-    {date: "Oct 20th", time: "14:45", away: "Scorpions", home: "Pina Chama"},
-    {date: "Oct 27th", time: "12:45", away: "Scorpions", home: "World Ventures"},
-    {date: "Oct 27th", time: "14:45", away: "Papa Spice", home: "Pina Chama"}
-]
 
-const htmlString = games
-  .map((game) => '' +
-    '<tr>' +
-      '<td>' + game.date + '</td>' +
-      '<td>' + game.time + '</td>' +
-      '<td class="' + colorOfTeam[game.away] + '">' + game.away + '</td>' +
-      '<td class="' + colorOfTeam[game.home] + '">' + game.home + '</td>' +
-    '</tr>'
-  )
-  .join('')
+const schedule = new XMLHttpRequest();
+schedule.onreadystatechange = function() {
+  if (schedule.readyState == 4 && schedule.status == 200) {
```

Comment:
The numbers `4` and `200` are what we call "magic numbers" – it's hard to understand their meaning without some additional context. A common way to give it meaning is to assign it to a variable (really, a `const`), e.g.:

```
// ...
const XHR_DONE = 4
const HTTP_OK = 200
if (schedule.readyState == XHR_DONE && schedule.status == HTTP_OK) {
  // ...
}
```

See how that could be a lot more readable?